src/movies.js

```diff
@@ -1,16 +1,179 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
+
+function getAllDirectors(movies) {
+    let dirArr = movies.map((i) => {
```

Comment:
the variable i is most used in loops, however, here, instead of i try to work around better names for the parameters of the methods, it will help a lot on bigger functions to understand what is going on. In this case something like movie or even eachMovie would be better.